src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -254,17 +254,21 @@
                                         :transaction-type   transaction-type}]
                                       (when (and (not loading-suggested-routes?) route (seq route))
                                         [standard-auth/slide-button
-                                         {:keycard-supported?  true
-                                          :size                :size-48
-                                          :track-text          (if (= transaction-type :tx/bridge)
-                                                                 (i18n/label :t/slide-to-bridge)
-                                                                 (i18n/label :t/slide-to-send))
-                                          :container-style     {:z-index 2}
+                                         {:keycard-supported? (get-in transaction-for-signing
+                                                                      [:signingDetails :signOnKeycard])
```

Comment:
If I understood correctly, this flag signifies whether the flow supports keycard signing or not (e.g. in swaps we would pass false, since it wasn't implemented yet). Whether keycard should actually be used or not is decided inside `standard-auth` events I think. @flexsurfer can you confirm?

Reply:
yes correct